docs/external-ip-and-loadbalancer-ingress-in-shared-gateway-mode.md

```diff
@@ -0,0 +1,45 @@
+# External IP and LoadBalancer Ingress in Shared Gateway mode
+
+OVN Kubernetes in Shared Gateway mode implements both External IPs and LoadBalancer Ingress IPs (`service.Status.LoadBalancer.Ingress`) in the form of OVN load balancers. These OVN load balancers live on all of the Kubernetes nodes and are thus highly available and ready for load sharing. A consequence of this highly available design is that OVN Kubernetes makes it the administrator's responsibility to route traffic to the Kubernetes nodes for both of these VIP types. 
+
+In an environment where External IPs and LoadBalancer Ingress VIPs happen to be part of the nodes' subnets, administrators might expect that OVN Kubernetes answer to ARP requests to these VIPs. However, this is not the case. The administrator is responsible for routing packets to the Kubernetes nodes and they cannot rely on the network plugin for this to happen.
+
+#### External IP
+
+The Kubernetes documentation states that External IPs are to be handled by the cluster administrator and are not the responsibility of the network plugin. See [https://kubernetes.io/docs/concepts/services-networking/service/#external-ips](https://kubernetes.io/docs/concepts/services-networking/service/#external-ips):
+~~~
+External IPs
+If there are external IPs that route to one or more cluster nodes, Kubernetes Services can be exposed on those externalIPs. Traffic that ingresses into the cluster with the external IP (as destination IP), on the Service port, will be routed to one of the Service endpoints. externalIPs are not managed by Kubernetes and are the responsibility of the cluster administrator.
```

Comment:
`that route to one or more cluster nodes`  I understand this phrase because I was involved in the discussions, hope its clear to others as well.

Reply:
It's a quote from the kubernetes doc. I rewrote this a bit, now it's:

#### External IP

The Kubernetes documentation states that External IPs are to be handled by the cluster administrator and are not the responsibility of the network plugin. See the following quote from the kubernetes documentation:
~~~
External IPs

If there are external IPs that route to one or more cluster nodes, Kubernetes Services can be exposed on those externalIPs. Traffic that ingresses into the cluster with the external IP (as destination IP), on the Service port, will be routed to one of the Service endpoints. externalIPs are not managed by Kubernetes and are the responsibility of the cluster administrator.
~~~
> Source: [https://kubernetes.io/docs/concepts/services-networking/service/#external-ips](https://kubernetes.io/docs/concepts/services-networking/service/#external-ips):